middlewares/createUserToken.js

```diff
@@ -0,0 +1,8 @@
+const jwt = require('jsonwebtoken')
+
+const { SECRET } = process.env
+
+exports.createUserToken = (user) => {
```

Comment:
Eli, para mi el "createUserToken" no es un middleware. No deberia estar en esta carpeta.
Deberias hacer:
1. ruta de login
2. controller de user con propiedad log in
3. validar que username y password del body sean los registrado en DB
4. si se valida el punto 3 --> creamos token a ese user y lo enviamos

Reply:
ok, lo puedo guardar en la carpeta service entonces